gallery/src/demos/demo-more-info-light.ts

```diff
@@ -32,28 +29,35 @@ const ENTITIES = [
   getEntity("light", "kitchen_light", "on", {
```

Comment:
Update this page to have an entity per supported color mode. That way it's easy to see if everything is supported.